test/e2e/apps/daemon_set.go

```diff
@@ -473,6 +478,9 @@ func newDaemonSet(dsName, image string, label map[string]string) *apps.DaemonSet
 			Name: dsName,
 		},
 		Spec: apps.DaemonSetSpec{
+			Selector: &metav1.LabelSelector{
```

Comment:
@foxish For `apps/v1`, there is a requirment to have `selector labels = template labels`.